mh/pat.py

```diff
@@ -134,6 +136,17 @@ def send_packet(self, packet_id=0, data=b'', seq=0):
             packet_id, seq, hexdump(data)
         )
 
+    def try_send_packet(self, packet_id=0, data=b'', seq=0):
+        """Send PAT packet and catch exceptions."""
+        try:
+            self.send_packet(packet_id, data, seq)
+        except Exception as e:
+            self.server.warning(
```

Comment:
I think a more useful information would be the exception message. Having the packet info doesn’t tell me anything what really happened.